command/stats_test.go

```diff
@@ -0,0 +1,89 @@
+package command_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/ghttp"
+	. "github.com/openebs/maya/command"
+	"os"
+)
+
+var _ = Describe("GetVolAnnotations", func() {
```

Comment:
Alright. The moment I start to review the stats_test.go code, I get lost trying to understand the ginkgo basics. That is my fault though. 

Somehow, i understand that GetVolAnnotations is the method or function under test. I looked at the src file i.e. stats.go & read the logic of GetVolAnnotations. It seems GetVolDetails call should be faked / mocked to unit test GetVolAnnotations. However, I am not able to understand the mock behaviour.